build_tools/cmake/iree_setup_toolchain.cmake

```diff
@@ -97,6 +97,14 @@ endif()
 if(IREE_ENABLE_ASAN)
   string(APPEND CMAKE_CXX_FLAGS " -fsanitize=address")
   string(APPEND CMAKE_C_FLAGS " -fsanitize=address")
+
+  # Not technically ASAN, but it guards against similar bugs in accessing
+  # uninitialized memory. See the extensive description in that patch that
+  # originally introduced it:
+  # https://reviews.llvm.org/rG14daa20be1ad89639ec209d969232d19cf698845
```

Comment:
>   - Integers are initialized with repeated 0xAA bytes (infinite scream).
>  - Vectors of integers are also initialized with infinite scream.
>  - Pointers are initialized with infinite scream on 64-bit platforms because
    it's an unmappable pointer value on architectures I'm aware of. Pointers
    are initialize to 0x000000AA (small scream) on 32-bit platforms because
    32-bit platforms don't consistently offer unmappable pages.

"infinite scream" and "small scream" :joy: